jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/SpringBootConfiguration.java

```diff
@@ -52,13 +53,23 @@ public static SpringBootConfiguration from(JavaProject project) {
       })
       // Defaults to Spring 1
       .orElse(1);
+    final int minorVersion = springBootVersion
+      .map(semVer -> {
+        try {
+          return Integer.parseInt(semVer.substring(semVer.indexOf('.'), semVer.lastIndexOf('.')));
+        } catch (Exception e) {
+          return null;
+        }
+      })
+      // Defaults to Spring 1
+      .orElse(1);
```

Comment:
This block doesn't look like it fits well here in this class. I see there is a class named [SemanticVersionUtil](https://github.com/eclipse-jkube/jkube/blob/master/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/SemanticVersionUtil.java) to handle semantic versions. Do you think we can move it there?

Reply:
done.